specs/models/Flow.schema.json

```diff
@@ -40,178 +43,195 @@
-                  "type": "boolean",
-                  "description": "If true, the query is applied to the documents that were created or modified in the same job that executes the step"
-                },
-                "provenanceGranularityLevel": {
-                  "type": "string",
-                  "description": "The granularity of the provenance tracking information: coarse (default) to store document-level provenance information only, fine to store document-level and property-level provenance information, or off to disable provenance tracking in future job runs. Applies only to mapping, matching, merging, mastering, and custom steps."
-                },
-                "stepUpdate": {
-                  "type": "boolean",
-                  "description": "If true, custom modules can make changes directly to records in the database"
-                },
-                "acceptsBatch": {
-                  "type": "boolean",
-                  "description": "If true, the step module is invoked once with all records in the batch passed to it"
+                  "description": "Name of step, though the 'step number' is how a step is referred to when running a flow"
                 },
-                "collections": {
-                  "type": "array",
-                  "items" : {
-                    "type": "string"
-                  }
-                },
-                "additionalCollections": {
-                  "type": "array",
-                  "items" : {
-                    "type": "string"
-                  }
-                },
-                "permissions": {
+                "description": {
                   "type": "string",
-                  "description": "Comma-delimited string of role,capability,role,capability,etc"
+                  "description": "Optional description of the step"
                 },
-                "outputFormat": {
+                "stepDefinitionName": {
                   "type": "string"
                 },
-                "sourceDatabase": {
+                "stepDefinitionType": {
                   "type": "string"
                 },
-                "targetDatabase": {
-                  "type": "string"
+                "batchSize": {
+                  "type": "number",
+                  "description": "If set, overrides the batchSize defined at the flow level and in the step definition"
                 },
-                "targetEntity": {
-                  "type": "string",
-                  "description": "Name of the entity type associated with the output of the step"
+                "threadCount": {
+                  "type": "number",
+                  "description": "If set, overrides the threadCount defined at the flow level and in the step definition"
                 },
-                "validateEntity": {
-                  "type": "boolean",
-                  "description": "Applicable to mapping steps only"
+                "retryLimit": {
```

Comment:
I included this at first to document it, but now I'm thinking we should remove it.

Reply:
removed